src/test/java/rx/plugins/RxJavaPluginsTest.java

```diff
@@ -108,6 +108,14 @@ public void testObservableExecutionHookViaRegisterMethod() {
     }
 
     @Test
+    public void testSingleExecutionHookViaRegisterMethod() {
```

Comment:
Right, but `@Test` clearly shows that it's a test, I hope we'll have some kind of codestyle document, IDE settings and checkstyle rules in future